pkg/registry/componentstatus/rest.go

```diff
@@ -85,11 +85,9 @@ func ToConditionStatus(s probe.Result) api.ConditionStatus {
 func (rs *REST) getComponentStatus(name string, server apiserver.Server) *api.ComponentStatus {
 	transport := rs.rt
 	status, msg, err := server.DoServerCheck(transport)
-	var errorMsg string
+	errorMsg := ""
 	if err != nil {
 		errorMsg = err.Error()
-	} else {
-		errorMsg = "nil"
```

Comment:
Basically bring up a new cluster and ensure that it works. The "validate-cluster.sh" script is invoked as part of "kube-up.sh". 

I see that tests have passed, so I think you're in the clear, but wanted to confirm if you've manually checked.